helm/istio/charts/kiali/templates/kiali-cr.yaml

```diff
@@ -8,14 +8,10 @@ spec:
   istio_namespace: "{{ .Release.Namespace }}"
 
   auth:
-{{- if .Values.dashboard }}
```

Comment:
The only change here should be adding `.Values.dashboard.secretName` to the `and` statement below.  If not all of those are set, we should be using oauth (`openshift`) by default.